### PR TITLE
Added 2014 Warren Ohio Precincts

### PR DESCRIPTION
**Pull request by hodgesmr**
Relevant to #7:

I used [pyesridump](https://github.com/openaddresses/pyesridump) to pull the Warren County, Ohio precinct geojson from [Warren County](http://webmap1.co.warren.oh.us/ArcGIS/rest/services/Test/BOE_November2014/MapServer/0). I then loaded that into QGIS to generate the shapefiles.

Two things:
- It's labeled as 2014. I didn't see anything labeled as 2016. Unsure if they changed.
- I noticed the directory is `../Test/..`, not sure if we care?

Also, please let me know if I should structure the files differently for the PR.

### CODE REVIEW

**Comment by nvkelso** [comment]
Nice work! I’m on phone at moment so will need to check details on Test later, but shouldn’t be problem.

The next step is to also modify the big Makefile in the Ohio section to exclude these counties from the census source, and add them from these 2 new sources. This is done for many other states so scan up and down for pattern, or we can help you out. 

Thank you!

**Comment by hodgesmr** [comment]
Great, I'll take a look at the Makefile.

I've also emailed a "GIS Technician" at the county to see if there are 2016 files available.

**Comment by jmillermi** [comment]
The precinct names in the statewide election results file are identical for 2014 and 2016, which *usually* means the precincts haven't changed.

**Comment by nvkelso** [comment]
My experience is any updates post 2010 redistricting (eg precincts from 2012) are a fairly stable baseline in most low- and medium-growth areas.

**Comment by hodgesmr** [comment]
The Makefile changes I've pushed denote these precincts as `2014`.

File: Makefile [hunk 1]
```diff
@@ -731,6 +732,10 @@ out/39-ohio/state.gpkg: data/39-ohio/statewide/2010/tl_2012_39_vtd10.zip \
 	ogr2ogr -sql "SELECT '2016' AS year, '39' AS state, '157' AS county, CONCAT('39157', CAST(PRCT_NAME AS character(30))) AS precinct, 'polygon' AS accuracy FROM TUS_PREC" \
 		-s_srs '+proj=lcc +lat_1=40.43333333333333 +lat_2=41.7 +lat_0=39.66666666666666 +lon_0=-82.5 +x_0=600000 +y_0=0 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=us-ft +no_defs' \
 		-t_srs EPSG:4326 -nln state -append -f GPKG $@ 'out/39-ohio/source/TUS_PREC.shp'
+	unzip -d out/39-ohio/source data/39-ohio/39165-warren/WarrenCountyOHPrecincts_2014.zip
+	ogr2ogr -sql "SELECT '2014' AS year, '39' AS state, '165' AS county, CONCAT('39165', CAST(prec_name AS character(30))) AS precinct, 'polygon' AS accuracy FROM WarrenCountyOHPrecincts_2014" \
+		-s_srs '+proj=lcc +lat_1=40.43333333333333 +lat_2=41.7 +lat_0=39.66666666666666 +lon_0=-82.5 +x_0=600000 +y_0=0 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=us-ft +no_defs' \
```
Comment (nvkelso) [inline Makefile]: QGIS reports this source as already being WGS 84 (`EPSG:4326`) – and including this source SRS results in a nation run that doesn't include the county / in the right spot. Suggest removing this line, or replacing with `EPSG:4326`

**Comment by hodgesmr** [comment]
The GIS Technician from the county provided me with the most recent precinct shapefiles (not the files provided in this PR). Those new files include a couple precincts that do no have any 2016 results, whereas the precincts in this PR _do_ all have results from 2016.

Two questions:

1. Should the layers in this PR be considered 2016 as opposed to 2014?
2. Is there a desire for the new shapefiles I received as a PR?

**Comment by nvkelso** [comment]
The new data and Makefile changes in this PR are working for me now.

As to the 2014 versus ~2016 data: Yes - the more data the merrier. Please submit a new PR with the ~2016 data and we can use that to discuss & document the miss-matches.

**Pull request merged**